lms/djangoapps/certificates/management/commands/rescind_cert.py

```diff
@@ -0,0 +1,104 @@
+"""
+Management command to update the status of a single user certificate
+for a given user, in a given course. If no --status kwarg is provided,
+the status will be updated to CertificiateStatuses.unavailable.
+"""
+import logging
+from optparse import make_option
+
+from django.contrib.auth.models import User
+from django.core.management.base import BaseCommand, CommandError
+from opaque_keys import InvalidKeyError
+from opaque_keys.edx.keys import CourseKey
+from opaque_keys.edx.locations import SlashSeparatedCourseKey
+
+from certificates.models import GeneratedCertificate, CertificateStatuses
+
+LOGGER = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    '''Update the status of a single user certificate for a given user in a given course'''
+    help = u'Rescind a certificate for a particular user in a particular course.'
+
+    option_list = BaseCommand.option_list + (
+        make_option(
+            '-c', '--course',
+            metavar='COURSE_ID',
+            dest='course',
+            default=False,
+            help='The course id (e.g., mit/6-002x/circuits-and-electronics) of the course in which the '
+                 'certificate of the student should be updated',
+        ),
+        make_option(
+            '-u', '--user',
+            metavar='USERNAME',
+            dest='username',
+            default=False,
+            help='The username or email address for whom grading and certificate status should be updated',
+        ),
+        make_option(
+            '-s', '--status',
+            metavar='STATUS',
+            dest='status',
+            default=CertificateStatuses.unavailable,
+            help='The status to set to the corresponding certificate',
+        ),
+    )
+
+    def handle(self, *args, **options):
+
+        if options['course']:
+            # try to parse out the course from the serialized form
+            try:
+                course_id = CourseKey.from_string(options['course'])
+            except InvalidKeyError:
+                course_id = SlashSeparatedCourseKey.from_deprecated_string(options['course'])
+        else:
+            raise CommandError(u'You must specify a course')
+
+        user = options['username']
+        if not user:
+            raise CommandError(u'The student\'s username is required')
+
+        status = options['status']
+        valid_statuses = [
+            CertificateStatuses.unavailable,
+            CertificateStatuses.downloadable,
+            CertificateStatuses.notpassing,
+        ]
+        if status not in valid_statuses:
```

Comment:
@caesar2164 There really is no such thing as _removing_ a cert; you can only hope to mark it as `unavailable`.

I think the idea here is that since `CertificateStatuses` is essentially an enum, we don't want to accidentally introduce "invalid" strings to the database. Which is to say, `status` should never be allowed to be set to `bogus`.

Please correct any improper assumptions, @azimpradhan